### ISSUE TITLE
Unable to view preview

### ISSUE THREAD

**Issue by iampapagray** [resend/react-email#1815]
### Describe the Bug

I have a NextJs project and i want to integrate react-email. Thus, i use the Manual Setup option in the docs from step 2 to the end. Because i already have a script with dev, i use `email: email dev --dir src/emails` in the package.json script.

When i open the preview page on localhost:3001 and navigate to the only email i added from the docs, i get this error:

`
Error: Build failed with 4 errors: emails/my-email.tsx:1:29: ERROR: Could not resolve "@react-email/components" emails/my-email.tsx:7:6: ERROR: Could not resolve "react/jsx-runtime" emails/my-email.tsx:17:33: ERROR: [plugin: rendering-utilities-exporter] Failed trying to import `render` from either `@react-email/render` or `@react-email/components` to be able to render your email template. Maybe you don't have either of them installed? emails/my-email.tsx:18:72: ERROR: Could not resolve "react"
`

Even after reinstalling the libraries it still doesnt work. 
I also tried the manual setup from scratch, exactly as it is in the docs and i get the same problem.

![Screenshot 2024-12-12 at 22 44 25](https://github.com/user-attachments/assets/67006490-7761-4c78-bc24-da213395777c)

Here are my env details:
MacOs 15.1.1
M3 Pro Chip
Node v21.6.2
NPM 10.2.4
bun 1.0.26

### Link to the code that reproduces this issue

none

### To Reproduce

simply follow steps here: https://react.email/docs/getting-started/manual-setup

### Expected Behavior

Preview for selected email should be displayed without a hustle.

### What's your node version? (if relevant)

21.6.2

**Comment by mjacquin** [comment]
Same issue here 😞 
Any advices ?

**Comment by iampapagray** [comment]
@mjacquin Nothing so far. I'm stuck

**Comment by haase1020** [comment]
> Many of the times I saw this error, while having all the packages installed, it was always because of an unwanted `.pnp.cjs` file at some place higher up from the project. Try looking up from the project, one directory at a time, while looking for a `.pnp.cjs` if there is one, delete it.

This did the trick for me. Thank you @gabrielmfern!

**Comment by gabrielmfern** [comment]
Everyone go for my fix above, if your case is not the one I mentioned, share a minimal reproduction, and I'll check it out.

**Comment by mjacquin** [comment]
> Many of the times I saw this error, while having all the packages installed, it was always because of an unwanted `.pnp.cjs` file at some place higher up from the project. Try looking up from the project, one directory at a time, while looking for a `.pnp.cjs` if there is one, delete it.

Worked for me !
The `.pnp.cjs` file was in my home directory.

Thanks @gabrielmfern

**Comment by iampapagray** [comment]
@gabrielmfern Sorry about the delayed feedback. I can't find this file you mentioned in my project. I've searched both manually and with IDE and system search but nothing came up.

**Comment by gabrielmfern** [comment]
@iampapagray not 100% sure all kinds of system search will find it, do it as I mentioned, and you're probably going to find it. Your IDE is also not going to detect it, since it's most likely in a directory higher up from your project, as I mentioned. 

But still, if you're in doubt of whether this is the cause or not, you can just test the same project on another machine. If the SAME error persists, then we can be sure it isn't caused by the `.pnp.cjs` file.

**Comment by gabrielgrs** [comment]
I am having the same issue, tried to find the pnp file, didn't find. Tried to create a project from zero, still having the same issue.

**Comment by iampapagray** [comment]
I am still facing the same issue. Can't find the ````.pnp.cjs```` file. @gabrielgrs Are you using bun?
@gabrielmfern is your environment similar to the one i described above?

**Comment by gabrielgrs** [comment]
> I am still facing the same issue. Can't find the `.pnp.cjs` file. [@gabrielgrs](https://github.com/gabrielgrs) Are you using bun? [@gabrielmfern](https://github.com/gabrielmfern) is your environment similar to the one i described above?

Yes, but also tried using npm and yarn and still having the same problems.

**Comment by djesmond1015** [comment]
I'm also facing the same [issue](https://github.com/resend/react-email/issues/1908) but on WSL Ubuntu.
However, it works on Windows 11.

**Comment by gabrielmfern** [comment]
Is everyone having this issue on WSL like @djesmond1015? It could be a specific issue there

**Comment by iampapagray** [comment]
@gabrielmfern No I'm on MacOs
